config.json

```diff
@@ -38,6 +54,7 @@
     "provider": "local"
   },
   "vector_database": {
-    "provider": "pgvector"
+    "provider": "pgvector",
+    "collection": "vector_database"
```

Comment:
We purposefully do not specify a collection in the config because default behavior is for config variables to override env vars.